app/models/marc_fields/bound_with_note_for_access_panel.rb

```diff
@@ -0,0 +1,26 @@
+###
+#  MarcBoundWith class to return 590 fields that have a $c
+###
+class BoundWithNoteForAccessPanel < BoundWithNote
+  def values
+    extracted_fields.map do |field, subfields|
+      id = subfields.find { |subfield| subfield.code == 'c' }.value[/^(\d+)/]
+
+      { id: id, value: display_value(field, subfields) }
+    end
+  end
+
+  def to_partial_path
+    'marc_fields/bound_with_note_for_access_panel'
+  end
+
+  private
+
+  def extracted_fields
+    super.select { |field, _subfields| field['c'].present? }
+  end
+
+  def display_value(field, subfields)
+    safe_join subfields.reject { |subfield| subfield.code == 'c' }.map { |subfield| subfield_value(field, subfield) }, subfield_delimeter
```

Comment:
OK, this one is twisting my wig a bit. First we're rejecting all the subfields with a code of `c`, and then we're using `subfield_value` to return values... but that means `subfield_value`'s special behavior, as defined in this class's parent (`BoundWithNote`), as isn't getting triggered so this is ultimately going up the chain to `BoundWithNote`'s parent. Is that what's intended? I feel like this nuance might make a good addition to the class-level documentation for this new class.

Reply:
Asking the hard questions... I don't know why this is the display behavior in the access panel or why it's different than the normal place. This code replaced

https://github.com/sul-dlss/SearchWorks/pull/2647/files#diff-c6b37349f78dc1bef9f5073e68179f4fcd4e51388aa19d7051ce4637fbc7037fL5

🤷‍♂️

Reply:
@cbeer I suspect it's OK. Looking at the code it replaced, it looks like the intent may have been to extract the catkey (from the 'c' subfield) but then ignore that subfield when displaying the value since we're apparently special-case displaying the catkey as the ID.

Reply:
Sounds right; but I think the why and wherefore have been lost to history.